addons/placement/XEH_postInit.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+
+GVAR(previewVehicle) = objNull;
+GVAR(lastMouse) = getMousePosition;
+
+["ZEN_displayCuratorLoad", {
+    params ["_display"];
+    GVAR(previewDraw) = addMissionEventHandler ["Draw3D", {
+        call FUNC(updatePreview);
+    }];
+}] call CBA_fnc_addEventHandler;
+
+
+[{!isNull (getAssignedCuratorLogic player)}, {
```

Comment:
Wouldn't it be better to just add this EH to all curators on their init? Curators EH only fire when curator logic is local and it will be local to the player that controls it AFAIK.

This WUAE will stay there forever for most of the players eating that micro seconds of execution time :P